### PR TITLE
Move integration tests to `/scripted-tests`

### PR DESCRIPTION
**Pull request by Duhemm**
They were in the resources, which means that sbt was copying them all
before running each tests, slowing down the whole thing. It also look
like sbt was looking at them to determine whether stuff was changed.

sbt feels a bit faster now.

### CODE REVIEW

**Comment by Duhemm** [comment]
test performance please

**Comment by bloopoid** [comment]
performance test scheduled: 1 job(s) in queue, 0 running.

**Comment by jvican** [comment]
A nitpick: can we call it `integration-tests` instead of `scripted-tests`?

**Comment by bloopoid** [comment]
Performance test finished successfully.

Benchmarks is based on merging with master

**Comment by Duhemm** [comment]
I changed that, let me re-check that the benchmarks still pas:

test performance please

**Comment by Duhemm** [comment]
test performance please

**Comment by bloopoid** [comment]
performance test scheduled: 1 job(s) in queue, 0 running.

**Comment by bloopoid** [comment]
Performance test finished successfully.

Benchmarks is based on merging with master

File: frontend/src/test/scala/bloop/tasks/IntegrationTestSuite.scala [hunk 1]
```diff
@@ -9,10 +9,7 @@ import bloop.{DynTest, Project}
 import bloop.io.AbsolutePath
 
 object IntegrationTestSuite extends DynTest {
-  val projects = Files.list(getClass.getClassLoader.getResources("projects") match {
-    case res if res.hasMoreElements => Paths.get(res.nextElement.getFile)
-    case _ => throw new Exception("No projects to test?")
-  })
+  val projects = Files.list(ProjectHelpers.testProjectsBase)
```
Comment (jvican) [inline frontend/src/test/scala/bloop/tasks/IntegrationTestSuite.scala]: I think this collides with the other PR, but we can fix it later on

**Pull request merged**